### PR TITLE
feat: basic quizs front

### PR DESCRIPTION
**Pull request by kfleury**
#### Description

Fixes #<issue number>

#### Changelog

- [BUGFIX]
- [ENHANCEMENT]
- [DOCS]
- ...

### CODE REVIEW

File: src/panels/QuizsPanels.tsx [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+    props.goBack()
+    return <></>
+  }
+
+  // Fetch current question
+  const currentQuestion = quiz?.questions?.[index]
+  if (currentQuestion == null) {
+    return (
+      <div className='my-4 text-center text-sm text-gray-400'>
+        <p>{'No quiz found'}</p>
```
Comment (kfleury) [inline src/panels/QuizsPanels.tsx]: Trad

File: src/panels/QuizsPanels.tsx [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+const ListQuizs: React.FC<{goBack: () => void, chooseQuiz: (_:V1Quiz) => void }> = (props) => {
+  const noteId = useNoteIdFromUrl()
+  const chooseQuiz = props.chooseQuiz
+  // const [index, setIndex] = React.useState(0)
+
+  // Can find note ? 
+  if (noteId == null) {
+    return (
+      <div className='my-4 text-center text-sm text-gray-400'>
+        <p>{'Your current note haven\'t been found'}</p>
```
Comment (kfleury) [inline src/panels/QuizsPanels.tsx]: Trad

**Comment by kfleury** [comment]
Tell me when you put the button

File: src/components/view/PanelHeader.tsx [hunk 3]
```diff
@@ -18,6 +19,7 @@ const PanelHeader: React.FC = () => {
   return (
     <div className='mt-xl flex h-[36px] min-h-[36px] items-center justify-around lg:mx-lg lg:mb-lg xl:mx-xl xl:mb-xl'>
       {panels.map((panelKey, idx) => {
+        // console.log(panelKey)
```
Comment (gabriel654165) [inline src/components/view/PanelHeader.tsx:22]: remove commentary please

File: src/hooks/api/recommendations.ts [hunk 4]
```diff
@@ -36,3 +36,34 @@ export const useGetWikipediaImage = (req: GetWikipediaImageRequest, options?: Qu
     ...options,
   })
 }
+
+export type GenerateQuizRequest = { noteId : string };
+export const useGenerateQuiz = (options?: MutationHookOptions<GenerateQuizRequest, V1GenerateQuizResponse>) => {
+  const authContext = useAuthContext()
+  const groupContext = useGroupContext()
+  const currentGroupId = groupContext.groupId as string
+  // const queryKey = newQuizsCacheKey(currentGroupId, req.noteId)
```
Comment (gabriel654165) [inline src/hooks/api/recommendations.ts:45]: Remove commentary please if you don't want to implem the cache

**Review by gabriel654165 — CHANGES_REQUESTED** [review]

**Review by gabriel654165 — APPROVED** [review]

**Pull request merged**